cli/src/commands/graph/monograph/commands/create.ts

```diff
@@ -43,7 +52,31 @@ export default (opts: BaseCommandOptions) => {
       }
     }
 
-    const spinner = ora('Federated Graph is being created...').start();
+    if (options.subscriptionProtocol && !isValidSubscriptionProtocol(options.subscriptionProtocol)) {
+      program.error(
+        pc.red(
+          pc.bold(
+            `The subscription protocol '${pc.bold(
+              options.subscriptionProtocol,
+            )}' is not valid. Please use one of the following: sse, sse_post, ws.`,
+          ),
+        ),
+      );
+    }
+
+    if (options.websocketSubprotocol && !isValidWebsocketSubprotocol(options.websocketSubprotocol)) {
```

Comment:
There should be not subprotocol when the subscriptionprotocol is not ws

Reply:
Done